components/event/EventCard.js

```diff
@@ -36,17 +36,23 @@ export default function EventCard({ event, username }) {
         </div>
         <div className="flex-1 space-y-1 p-4">
           <div className="flex items-center justify-between">
-            <div>
+            <Link
+              href={event.url}
+              key={event.url}
+              target="_blank"
+              rel="noreferrer"
+              className="text-decoration-line: none"
+            >
               <div className="flex justify-between">
-                <Link
+                <p
```

Comment:
I'm not sure why you used `href`, `key`, `target` ad `rel` to the `<p>`.
It needs to be removed,

Reply:
i have removed it